tutorials/Brier_Score.ipynb

```diff
@@ -1,131 +1,2206 @@
+                "- In the future, the Brier score components calculation will be added.\n",
+                "- You may be interested in working through the Murphy Diagram tutorial which allows you to break down the performance of the Brier score based on each threshold probability.\n",
+                "\n",
+                "## Using the Brier score to evaluate ensemble forecasts\n",
+                "\n"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Scores can also be used to evaluate ensembles using the Brier score. By default, it calculates the fair Brier score which is defined as\n",
+                "\n",
+                "$$s_{i,y} = \\left(\\frac{i}{m} - y\\right)^2 - \\frac{i(m-i)}{m^2(m-1)}$$\n",
+                "\n",
+                "Where:\n",
+                "\n",
+                "- $i$ is the number of ensemble members that predict the event\n",
+                "- $m$ is the total number of ensemble members\n",
+                "- $y$ is the observed event\n",
+                "\n",
+                "It is possible to calculate the Brier score without the fair correction (by setting ``fair_correction=False``). In this case, the Brier score is calculated as\n",
+                "\n",
+                "\n",
+                "$$s_{i,y} = \\left(\\frac{i}{m} - y\\right)^2$$\n",
+                "\n",
+                "The use of the Brier score without the fair adjustment may favour ensembles that have members that have the behaviour of being sampled from a different distribution than the observations. For more information, see [Ferro (2014)](https://doi.org/10.1002/qj.2270).\n",
+                "\n",
+                "Let's first do a synthetic experiment with 10,000 timesteps to demonstrate the impact of the fair correction using the scores function. Suppose the observations are randomly drawn from the distribution $\\mathrm{Pr}(y=1) = 0.5$. Let's now create two ensemble forecasts. The first (we will call `fcst1`) has 2 ensemble members where the value for each member is randomly drawn from the same distribution that the observations are drawn from and is an ideal forecast. The second ensemble (called `fcst2`) has 20 ensemble members but is biased. The value for each member is randomly drawn from the distribution $\\mathrm{Pr}(y=1) = 0.4$.\n",
+                "\n",
+                "Let's now calculate the Brier score with and without the fair adjustment."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 15,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "from scores.probability import brier_score_for_ensemble\n",
+                "\n",
+                "# Uncomment the following line to learn more about the ensemble Brier score\n",
+                "#help(ensemble_brier_score)\n",
```

Comment:
```suggestion
                "#help(brier_score_for_ensemble)\n",
```
Is this meant to be brier_score_for_ensemble?

Please note: because I reviewed this in the GitHub PR diff, and not in JupyterLab, I have not uncommented the commented out line. As such, I have not reviewed the commented out section.

Reply:
Yes  - good pickup